### PR TITLE
Changes the brain damage moodlet blurb to be easier to understand.

### PR DESCRIPTION
**Pull request by EmptyLullaby**
## About The Pull Request
I've been playing recently and noticed that I had a negative moodlet that I had no idea the source of, I checked the code and realized it was brain damage. I felt like it was worthwhile to change the moodlet blurb to make it easier to understand what is causing it.

Also I have not used github in a hot like, year, sorry if anything is off about this pr I'll remake it if needed.

## Why It's Good For The Game
Making causes for negative moods clear is good.

## Changelog
:cl:
qol: Changed the blurb for the brain damage moodlet to make it easier to understand.
/:cl:

### CODE REVIEW

**Comment by EmptyLullaby** [comment]
oh thats not what i titled that commit at all thats weird

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -113,7 +113,7 @@
 
 /datum/mood_event/brain_damage/add_effects()
 	var/damage_message = pick_list_replacements(BRAIN_DAMAGE_FILE, "brain_damage")
```
Comment (Mothblocks) [inline code/datums/mood_events/generic_negative_events.dm:115]: Remove this if you're not going to use it.

Comment (EmptyLullaby) [reply code/datums/mood_events/generic_negative_events.dm:115]: oopsie doopsies ill do that thank you

**Comment by YakumoChen** [comment]
The problem with the new line is that it doesn't really make sense from a game or even realism perspective. Brain damage isn't a quantifiable thing you're supposed to feel in game outside of its direct effects (being traumas).

People with brain damage often don't realize they _have_ brain damage. Saying "ow my brain hurts and it's hard to think" isn't immersive or even believable.

**Pull request closed without merging**